File: C#/WindowsApp6/Form1.cs
using System;
using System.Drawing;
using System.Windows.Forms;
using WindowsApp6.Seal;

namespace WindowsApp6
{

    public partial class Form1
    {
        private int x, y;
        private Point newpoint = new Point();

        public Form1()
        {
            InitializeComponent();
        }
        private void PictureBox1_Click(object sender, EventArgs e)
        {
            bool response = LOGIN.Seal.Login(TextBox1.Text, TextBox2.Text);

            if (response)
            {
                My.MySettingsProperty.Settings.user = TextBox1.Text;
                My.MySettingsProperty.Settings.pass = TextBox2.Text;
                My.MySettingsProperty.Settings.Save();
                My.MySettingsProperty.Settings.Reload();
                My.MyProject.Forms.Form2.Show();
                Hide();
            }

        }

        private void PictureBox2_Click(object sender, EventArgs e)
        {
            SealCheck.HashChecks();

            if (SealCheck.isValidDLL)
            {
                bool response = LOGIN.Seal.Register(TextBox1.Text, TextBox2.Text, "SHEHAB@GMAIL.COM", TextBox3.Text);

                if (response)
                {
                    // Register Success
                }
            }
        }

        private void PictureBox4_Click(object sender, EventArgs e)
        {
            TextBox3.Visible = false;
            Label3.Visible = false;
            PictureBox3.Visible = true;
            PictureBox4.Visible = false;

        }

        private void PictureBox3_Click(object sender, EventArgs e)
        {
            TextBox3.Visible = true;
            Label3.Visible = true;
            PictureBox3.Visible = false;
            PictureBox4.Visible = true;
        }

        private void Form1_MouseMove(object sender, MouseEventArgs e)
        {
            if (e.Button == MouseButtons.Left)
            {
                newpoint = MousePosition;
                newpoint.X -= x;
                newpoint.Y -= y;
                Location = newpoint;
                Application.DoEvents();
            }
        }

        private void Form1_MouseDown(object sender, MouseEventArgs e)
        {
            x = MousePosition.X - Location.X;
            y = MousePosition.Y - Location.Y;
        }



        private void Button1_Click(object sender, EventArgs e)
        {
            Close();
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            SealCheck.HashChecks();
            if (SealCheck.isValidDLL)
            {
                LOGIN.Seal.Secret = "Ly9Nepf07acVE3EDw9KuNPCdqngeNlIEZNwheXtz3GKkU";
                LOGIN.Seal.Initialize("1.0");
            }
            TextBox1.Text = My.MySettingsProperty.Settings.user;
            TextBox2.Text = My.MySettingsProperty.Settings.pass;
            TextBox3.Visible = false;
            Label3.Visible = false;


            PictureBox4.Visible = false;

        }
    }
}

File: C#/WindowsApp6/Form1.vb
Imports LOGIN
Imports TrinitySeal
Imports WindowsApp6.Seal

Public Class Form1
    Dim x, y As Integer
    Dim newpoint As New Point
    Private Sub PictureBox1_Click(sender As Object, e As EventArgs) Handles PictureBox1.Click
        Dim response As Boolean = LOGIN.Seal.Login(TextBox1.Text, TextBox2.Text)

        If response Then
            My.Settings.user = TextBox1.Text
            My.Settings.pass = TextBox2.Text
            My.Settings.Save()
            My.Settings.Reload()
            Form2.Show()
            Me.Hide()
        End If

    End Sub

    Private Sub PictureBox2_Click(sender As Object, e As EventArgs) Handles PictureBox2.Click
        SealCheck.HashChecks()

        If SealCheck.isValidDLL Then
            Dim response As Boolean = LOGIN.Seal.Register(TextBox1.Text, TextBox2.Text, "SHEHAB@GMAIL.COM", TextBox3.Text)

            If response Then
                'Register Success
            End If
        End If
    End Sub

    Private Sub PictureBox4_Click(sender As Object, e As EventArgs) Handles PictureBox4.Click
        TextBox3.Visible = False
        Label3.Visible = False
        PictureBox3.Visible = True
        PictureBox4.Visible = False

    End Sub

    Private Sub PictureBox3_Click(sender As Object, e As EventArgs) Handles PictureBox3.Click
        TextBox3.Visible = True
        Label3.Visible = True
        PictureBox3.Visible = False
        PictureBox4.Visible = True
    End Sub

    Private Sub Form1_MouseMove(sender As Object, e As MouseEventArgs) Handles MyBase.MouseMove
        If e.Button = Windows.Forms.MouseButtons.Left Then
            newpoint = Control.MousePosition
            newpoint.X -= x
            newpoint.Y -= y
            Me.Location = newpoint
            Application.DoEvents()
        End If
    End Sub

    Private Sub Form1_MouseDown(sender As Object, e As MouseEventArgs) Handles MyBase.MouseDown
        x = Control.MousePosition.X - Me.Location.X
        y = Control.MousePosition.Y - Me.Location.Y
    End Sub



    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        Close()
    End Sub

    Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        SealCheck.HashChecks()
        If SealCheck.isValidDLL Then
            LOGIN.Seal.Secret = "Ly9Nepf07acVE3EDw9KuNPCdqngeNlIEZNwheXtz3GKkU"
            LOGIN.Seal.Initialize("1.0")
        End If
        TextBox1.Text = My.Settings.user
        TextBox2.Text = My.Settings.pass
        TextBox3.Visible = False
        Label3.Visible = False


        PictureBox4.Visible = False

    End Sub
End Class